.github/workflows/fastapi.yml

```diff
@@ -0,0 +1,40 @@
+name: fastapi-testsuite
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  fastapi-test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: fastapi
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
```

Comment:
think I'll echo a previous comment, we can probably just test 1 version of Python, since FastAPI is testing that they are compatible with all of these versions, we just need to be sure we are compatible with FastAPI

(unless our instrumentation has a bunch of python version conditionals?)

Reply:
@brettlangdon Sounds good. I will change the conditional to the latest version (3.9)